src/row_codec.rs

```diff
@@ -1,182 +1,230 @@
-use crate::db_state::RowFeature;
 use crate::error::SlateDBError;
-use crate::types::{KeyValueDeletable, RowAttributes, ValueDeletable};
+use crate::types::{RowEntry, ValueDeletable};
 use bitflags::bitflags;
 use bytes::{Buf, BufMut, Bytes, BytesMut};
 
 bitflags! {
+    #[derive(Debug, Clone, PartialEq, Default)]
     pub(crate) struct RowFlags: u8 {
         const Tombstone = 0b00000001;
+        const HAS_EXPIRE_TS = 0b00000010;
+        const HAS_CREATE_TS = 0b00000100;
     }
 }
 
-const NO_EXPIRE_TS: i64 = i64::MIN;
-
 /// Encodes key and value using the binary codec for SlateDB row representation
-/// using the `v0` encoding scheme.
+/// using the `v1` encoding scheme.
```

Comment:
I think it's probably fine to keep the codec at v0 until we make a 1.0 release and stabilize the codec, the idea of having versions for the codec would be to require maintaining compatbility for each one so that if you bump the codec version new versions will still be able to read the old one. if we don't maintain compatibility there's no reason to bump the codec

Reply:
make sense :+1:, changed it back to v0 in 82753dc